### ISSUE TITLE
Can't converge Loss function 

### ISSUE THREAD

**Issue by eliwilner** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#431]
Hi,

great work! I have a quick question regarding an issue I have.

I've been using a great data source (https://github.com/pythonlessons/OIDv4_ToolKit) which enables you to download images from specific desired class. I'm using 6 classes (food, drink, animal, person, woman and man) with 800 images per class.
I'm running the training on aws EC2 instance but I can't seems to converge the loss function to <0.05. It seems to fluctuate a lot and I'm already at step~ 52k.

Could anyone advise why this is happening? Is it a problem with the dataset?

Thanks

**Comment by maacofficial** [comment]
> Hi,
> 
> great work! I have a quick question regarding an issue I have.
> 
> I've been using a great data source (https://github.com/pythonlessons/OIDv4_ToolKit) which enables you to download images from specific desired class. I'm using 6 classes (food, drink, animal, person, woman and man) with 800 images per class.
> I'm running the training on aws EC2 instance but I can't seems to converge the loss function to <0.05. It seems to fluctuate a lot and I'm already at step~ 52k.
> 
> Could anyone advise why this is happening? Is it a problem with the dataset?
> 
> Thanks

Virtual machine's performance is bad about tensorflow training. Also I have tried many vds and vps but they don't have performance about it. Because machines controlled by server virtualization software like vmware vsphere.

**Comment by maacofficial** [comment]
> Hi,
> 
> great work! I have a quick question regarding an issue I have.
> 
> I've been using a great data source (https://github.com/pythonlessons/OIDv4_ToolKit) which enables you to download images from specific desired class. I'm using 6 classes (food, drink, animal, person, woman and man) with 800 images per class.
> I'm running the training on aws EC2 instance but I can't seems to converge the loss function to <0.05. It seems to fluctuate a lot and I'm already at step~ 52k.
> 
> Could anyone advise why this is happening? Is it a problem with the dataset?
> 
> Thanks

Also tensorflow cpu is worse than tensorflow gpu.

**Comment by eliwilner** [comment]
> > Hi,
> > great work! I have a quick question regarding an issue I have.
> > I've been using a great data source (https://github.com/pythonlessons/OIDv4_ToolKit) which enables you to download images from specific desired class. I'm using 6 classes (food, drink, animal, person, woman and man) with 800 images per class.
> > I'm running the training on aws EC2 instance but I can't seems to converge the loss function to <0.05. It seems to fluctuate a lot and I'm already at step~ 52k.
> > Could anyone advise why this is happening? Is it a problem with the dataset?
> > Thanks
> 
> Virtual machine's performance is bad about tensorflow training. Also I have tried many vds and vps but they don't have performance about it. Because machines controlled by server virtualization software like vmware vsphere.

Hi,
Not sure I understand your response... Can you please elaborate? Why should I have problem training on an EC2 instance?

**Comment by maacofficial** [comment]
Do you use cpu for it?

**Comment by maacofficial** [comment]
> > > Hi,
> > > great work! I have a quick question regarding an issue I have.
> > > I've been using a great data source (https://github.com/pythonlessons/OIDv4_ToolKit) which enables you to download images from specific desired class. I'm using 6 classes (food, drink, animal, person, woman and man) with 800 images per class.
> > > I'm running the training on aws EC2 instance but I can't seems to converge the loss function to <0.05. It seems to fluctuate a lot and I'm already at step~ 52k.
> > > Could anyone advise why this is happening? Is it a problem with the dataset?
> > > Thanks
> > 
> > 
> > Virtual machine's performance is bad about tensorflow training. Also I have tried many vds and vps but they don't have performance about it. Because machines controlled by server virtualization software like vmware vsphere.
> 
> Hi,
> Not sure I understand your response... Can you please elaborate? Why should I have problem training on an EC2 instance?

If you use tensorflow cpu, pls change it to gpu.Also If you already use tensorflow gpu,it is standard and it is rare value that is less then 0.5. Usually I get same values too.
